scripts/run_tests_bayesian.py

```
@@ -0,0 +1,203 @@
+                test_context,
+                f"failed_test_{test_name}",
+                f"tests/{test_name}",
+            )
+
+            if result.critical_issues:
+                print(f"  ❌ Критические проблемы: {len(result.critical_issues)}")
+                for issue in result.critical_issues[:3]:
+                    print(f"     - {issue}")
+
+            if result.optimization_opportunities:
+                print(f"  💡 Возможности оптимизации:")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)